features/create_case.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Case Management
+@Basic
+Scenario: Create a case
```

Comment:
Since this test will be run many times, you should make sure the case is closed, otherwise even if case creation 'fails', the second part of the test will still succeed. Maybe we can just use the `create_update_close` instead of this test, since it encapsulates case creation?